install/streams.csv

```diff
@@ -1619,10 +1619,8 @@ PUZ,10,50,false,false,true,1989-03-08T11:22:00Z,2003-08-19T12:44:00Z
 PUZ,10,100,false,false,false,2003-08-20T00:00:00Z,9999-01-01T00:00:00Z
 PUZ,11,50,false,false,true,2003-08-21T07:30:00Z,2003-12-16T06:27:00Z
 PUZ,20,1,false,false,false,2003-08-20T00:00:00Z,2013-08-07T00:00:00Z
-PUZ,20,50,false,false,false,2013-08-07T00:05:00Z,2019-11-21T22:53:00Z
-PUZ,20,200,false,false,true,2003-08-20T00:00:00Z,2019-11-21T22:53:00Z
-PUZ,21,50,false,false,false,2019-11-21T22:55:00Z,2020-06-16T00:00:00Z
-PUZ,21,200,false,false,true,2019-11-21T22:55:00Z,9999-01-01T00:00:00Z
+PUZ,21,50,false,false,false,2013-08-07T00:05:00Z,2020-06-16T00:00:00Z
+PUZ,21,200,false,false,true,2003-08-20T00:00:00Z,9999-01-01T00:00:00Z
```

Comment:
open the 200hz continuous on Feb 9 and close the 200hz triggered on Feb 8?

Reply:
will  do :0